### PR TITLE
Implement CodeSandbox for the examples

### PR DESCRIPTION
**Pull request by CompuIves**
I created a new library called `@codesandbox/react-embed`. This is a simple component that will create a sandbox on mount using our API and render an embed which shows the sandbox.

I had to change two examples to make them work:

- Animation
	- I import `styles.css` now
- QueryParams
	- I wrap the component in a Route, because `location` is needed by the component.

### CODE REVIEW

**Comment by pshrmn** [comment]
Looks great!

rr-sandbox

**Comment by CompuIves** [comment]
Awesome! Animation example does not work yet, because it tries to download `styles.css` from the GitHub repo which isn't on master yet. This happens for files outside of the main example.

**Comment by CompuIves** [comment]
It should work when it's merged to `master` though, forgot to mention that. This one should be mergeable if it's LGTM!

File: website/modules/components/WebExample.js [hunk 1]
```diff
@@ -1,57 +1,37 @@
 import React from "react";
 import PropTypes from "prop-types";
-import Media from "react-media";
-import { Block } from "jsxstyle";
-import { Route } from "react-router-dom";
-import Bundle from "./Bundle";
-import FakeBrowser from "./FakeBrowser";
```
Comment (pshrmn) [inline website/modules/components/WebExample.js]: I don't think that this module is used anywhere, would you mind removing it?

**Comment by CompuIves** [comment]
Good one! Pushed it.

**Comment by mjackson** [comment]
Thanks for the work here, @CompuIves. I love that you have a React component for embedding a Codesandbox example on a web page 🔥

I wonder if we could we use react-native-web or something similar to run our `react-router-native` examples on Codesandbox as well...

**Comment by CompuIves** [comment]
> I wonder if we could we use react-native-web or something similar to run our react-router-native examples on Codesandbox as well...

Seems like it works out of the box! I moved the native examples as well now.

**Comment by CompuIves** [comment]
screenshot 2018-12-02 at 03 12 25

**Comment by CompuIves** [comment]
Heyo! Just wanted to make sure that the PR wouldn't go stale. Are there more required changes to merge this?

**Comment by bradwestfall** [comment]
@CompuIves I'm going to look at this today and see if we can merge. Sorry for the delay

**Pull request merged**